shell/linters/prettier.sh

```diff
@@ -10,6 +10,13 @@ source "$DIR/languages/nodejs.sh"
 extensions=(yaml yml json md ts)
 
 PRETTIER="node_modules/.bin/prettier"
+if [[ ! -f $PRETTIER ]] && [[ ! -f package.json ]]; then
+  # Try to find prettier installed via mise
+  PRETTIER="$(mise which prettier)"
+  if [[ -z $PRETTIER ]]; then
+    fatal "prettier not found in repo"
```

Comment:
Should we update wording of the message? Current one will cause people to write to #dev-tooling-support, maybe we can prompt them to install prettier via `mise`?

Reply:
```suggestion
    fatal "prettier not found in repo, make sure 'npx:prettier' is defined in 'mise.toml' and you have run 'mise install'"
```